### PR TITLE
Changed num files of some M benchmarks to lower tolerances.

### PR DESCRIPTION
**Pull request by nagelt**
This PR is associated with benchmarks that fail due to PR Minkley_Burgers_Fixes because of new convergence settings for deformation problems. Num file settings are adjusted here.

### CODE REVIEW

**Comment by wenqing** [comment]
:+1:

File: TH2M/Newton/note_on_benchmark.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Tolerance settings very high. Physical convergence not guaranteed. Benchmark does not test solution but basic functioning of TH2M Newton loop.
```
Comment (norihiro-w) [inline TH2M/Newton/note_on_benchmark.txt]: what is a reason behind the tolerance setting? if this benchmark cannot provide correct solution, it should be deactivated until it's fixed.

Comment (norihiro-w) [inline TH2M/Newton/note_on_benchmark.txt]: Note) The deactivation is not possible in this repository. we need to modify a CMake file in the source code repository.

**Comment by norihiro-w** [comment]
@nagelt is it ready for merging?

**Comment by wenqing** [comment]
:fast_forward:

**Comment by nagelt** [comment]
@norihiro-w yes

**Pull request merged**